### PR TITLE
Bump patternfly4 and patternfly react-core

### PR DESCRIPTION
**Pull request by sg00dwin**

### CODE REVIEW

**Comment by spadgett** [comment]
Looks like we need to update our tests that check buttons after https://github.com/patternfly/patternfly-react/pull/2710

**Comment by alecmerdler** [comment]
Added commit to fix the unit tests.

File: frontend/packages/dev-console/src/components/progressive-list/__tests__/ProgressiveListFooter.spec.tsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import * as React from 'react';
 import { shallow } from 'enzyme';
+import { Button } from '@patternfly/react-core';
+
```
Comment (spadgett) [inline frontend/packages/dev-console/src/components/progressive-list/__tests__/ProgressiveListFooter.spec.tsx]: the empty line here is causing the lint error

**Pull request merged**